application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue

```diff
@@ -241,6 +246,8 @@ export default {
       installExtensions(userSettings);
       this.composerApplications = composerApplications;
     });
+    const currentUserIdentityId = eXo.env.portal.profileOwnerIdentityId;
+    this.$identityService.getIdentityById(currentUserIdentityId).then( user => this.currentUser = user );
```

Comment:
you can use directly that code instead
`this.$userService.getUser(eXo.env.portal.userName)
        .then(user => {
          this.isExternal = user.external === 'true';
        });
    }`